### ISSUE TITLE
Prepare of CREATE TABLE and INSERT fails without error details

### ISSUE THREAD

**Issue by paurkedal** [andrenth/ocaml-mariadb#1]
The command
```sh
OCAML_MARIADB_QUERY='CREATE TABLE test (i INTEGER)' ./blocking_example.native
```
gives
```
Fatal error: exception Failure("prepare: (0) ")
```
The query `INSERT INTO test VALUES (1)` gives the same error, so it's probably related to statements returning empty results.

**Comment by andrenth** [comment]
Hi.

Thanks for the report. I've pushed a fix to the master branch that I believe fixes this. Could you try it out?

It required an API change (`execute` returns an `Res.t option result` to account for an `Ok` case with no results).

**Comment by paurkedal** [comment]
It works, thanks!

**Comment by andrenth** [comment]
I've published a new version, 0.6.0. It should be available on OPAM soon.

Cheers!
